cmd/main.go

```diff
@@ -38,7 +38,7 @@ func main() {
 			&pluginv2.Plugin{},
 		),
 		cli.WithDefaultPlugins(config.Version3Alpha,
-			&pluginv2.Plugin{},
```

Comment:
Are we ok with not allow projects v2 to be scaffolded in the PROJECT version v3-alpha? 
Is not it a breaking change for SDK tool side @estroz ?

Reply:
This change is only setting the default plugin for a project version, not removing support for 3-alpha in go/v2.

Reply:
It shows not part of the scope of this PR. Could we revert it?

Reply:
This change doesn't do it but the first commit does. I'm still on working on supporting `3-alpha` in `go/v2`.

Reply:
If this gets reverted, tests should pass.